radio/adaptors/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can we move the copyrights to the new year :-)

Reply:
Good point!